components/ws-daemon/pkg/quota/xfs.go

```diff
@@ -117,7 +117,13 @@ func (xfs *XFS) SetQuota(path string, quota Size) (projectID int, err error) {
 	if err != nil {
 		return 0, err
 	}
-	_, err = xfs.exec(xfs.Dir, fmt.Sprintf("limit -p bsoft=%d bhard=%d %d", quota, quota, prjID))
+
+	if isHard {
+		_, err = xfs.exec(xfs.Dir, fmt.Sprintf("limit -p bhard=%d %d", quota, prjID))
```

Comment:
@utam0k prior to making this change, were you able to add content to `/workspace` and exceed the quota?

Also, if the quota is exceeded in the regular workspace, what happens now to users if they try writing more content to `/workspace`, but at are the hard limit?